tests/golden/main.rs

```diff
@@ -0,0 +1,46 @@
+use std::{
+    fs,
+    path::{Path, PathBuf},
+};
+
+use assert_cmd::Command;
+use assert_fs::{assert::PathAssert, prelude::PathChild};
+
+fn fixtures_path() -> PathBuf {
+    Path::new(env!("CARGO_MANIFEST_DIR"))
+        .join("tests")
+        .join("golden")
+        .join("fixtures")
+}
+
+#[test]
+fn golden() {
```

Comment:
here is the test i mentioned in the previous comment :) , so perhaps run it on the workflows by using the action too?

Reply:
Yes! This is implemented in the second PR: https://github.com/newrelic/rust-licenses-noticer/pull/2

I can merge it into this one, but I wanted to keep it separate because it rewrote the code from its previous state in AC, so it was easier to review.